api/src/test/java/org/openmrs/module/ugandaemrreports/StandaloneContextSensitiveTest.java

```diff
@@ -22,15 +22,13 @@ public Boolean useInMemoryDatabase() {
 	 */
 	@Override
 	public Properties getRuntimeProperties() {
-		Properties p = super.getRuntimeProperties();
+		System.setProperty("databaseUrl", "jdbc:mysql://localhost:3306/openmrs?autoReconnect=true&sessionVariables=storage_engine%3DInnoDB&useUnicode=true&characterEncoding=UTF-8");
+		System.setProperty("databaseUsername", "openmrs");
```

Comment:
@slubwama this method as you see it and also override the useInMemoryDatabase() method to return false.